### DISCUSSION TITLE
UIs built on fairyGUI tool are breaking on HD and SD devices

### DISCUSSION THREAD

**Discussion by iAbdulQadirKhan** [axmolengine/axmol#1594] [General]
All our UIs built on fairyGUI tool are breaking on HD and SD devices. But working fine with HDR devices. It was working fine on **cocos-2dx-v3**. 

**Expected:**
![Screenshot_20240110_162439_Cleopatra Slots](https://github.com/axmolengine/axmol/assets/79637143/2ed5373b-e5b0-4f29-a4e1-71002f7cddb3)

**Actual:** 
![Screenshot_20240110-162245](https://github.com/axmolengine/axmol/assets/79637143/a8d1e2ac-8957-4cfa-b7f5-fdd6bf92f3b6)

**Comment by iAbdulQadirKhan** [comment]
We have fixed this issue by making these changes in **UIPackage.cpp** file.

<img width="529" alt="Screenshot 2024-01-11 at 1 43 36 PM" src="https://github.com/axmolengine/axmol/assets/79637143/512a15f6-1fa1-4896-92f4-554968776af8">

**Reply by halx99** [reply]
Nice, could you create a pull-request?
